cmd/pj-rehearse/main.go

```diff
@@ -289,8 +289,9 @@ func rehearseMain() error {
 	changedPresubmits := diffs.GetChangedPresubmits(masterConfig.Prow, prConfig.Prow, logger)
 	toRehearse.AddAll(changedPresubmits, config.ChangedPresubmit)
 
-	presubmitsForCiopConfigs := diffs.GetPresubmitsForCiopConfigs(prConfig.Prow, changedCiopConfigData, affectedJobs, logger)
+	presubmitsForCiopConfigs, periodicsForCiopConfigs := diffs.GetJobsForCiopConfigs(prConfig.Prow, changedCiopConfigData, affectedJobs, logger)
 	toRehearse.AddAll(presubmitsForCiopConfigs, config.ChangedCiopConfig)
+	periodicsToRehearse.AddAll(periodicsForCiopConfigs)
```

Comment:
Why does this not take the `config.ChangedCiopConfig` reason like the presubmits do?

Reply:
That's... a good question. I did not touch *that* code and just did what other sources of rehearsed periodics do. Let me look into this.